webapp/components/anomalies.html

```diff
@@ -0,0 +1,143 @@
+<!--
+Copyright 2017 The WPT Dashboard Project. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+-->
+
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-if.html">
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-repeat.html">
+<link rel="import" href="runs.html">
+<link rel="import" href="test-file-results.html">
+<link rel="import" href="test-run.html">
+
+<dom-module id="wpt-anomalies">
+  <template>
+    <style>
+    </style>
+
+    <section class="caveat">
+      Data below are intended for web platform implementers and do not contain useful metrics for evaluation or comparison of web platform features. Also note that tested Edge and Safari are not pre-release versions (<a href="https://github.com/w3c/wptdashboard/issues/109">#109</a>, <a href="https://github.com/w3c/wptdashboard/issues/110">#110</a>).
+    </section>
+
+    <template is="dom-repeat" items="{{anomalyGroups}}" as="anomalyGroup" itemsIndexAs="index">
+      <template is="dom-if" if="{{browser}}">
+        <h2>Failing in [[browser]] and [[computeOtherBrowserCount(index)]]</h2>
+      </template>
+      <template is="dom-if" if="{{!browser}}">
+        <h2>Passing in [[computeBrowserCount(anomalyGroup.passingBrowsers)]]</h2>
```

Comment:
I would've deferred pluralization semantics to a third party library. That can be done in a follow up

Reply:
Opened https://github.com/w3c/wptdashboard/issues/379 to track this. Did you have a specific library in mind?